include/ck/tensor_operation/gpu/grid/gridwise_gemm_bias_add_reduce_xdl_cshuffle_v1.hpp

```diff
@@ -23,19 +23,19 @@ template <typename GridwiseGemm,
           typename FloatC,
           typename FloatC0,
           typename FloatC1,
-          typename DPtrsGlobal,
+          typename ReducePtrsGlobal,
```

Comment:
again. naming: "D" is more concise, and would be clearer than "Reduce" if you define what this Device op does by writing down the formula in comment